src/main/java/org/fenixedu/academic/json/adapters/AdministrativeOfficeJsonAdapter.java

```diff
@@ -0,0 +1,72 @@
+package org.fenixedu.academic.json.adapters;
+
+import org.fenixedu.academic.domain.accounting.serviceAgreementTemplates.AdministrativeOfficeServiceAgreementTemplate;
+import org.fenixedu.academic.domain.administrativeOffice.AdministrativeOffice;
+import org.fenixedu.academic.domain.administrativeOffice.AdministrativeOfficeType;
+import org.fenixedu.academic.domain.organizationalStructure.AccountabilityType;
+import org.fenixedu.academic.domain.organizationalStructure.AccountabilityTypeEnum;
+import org.fenixedu.academic.domain.organizationalStructure.AggregateUnit;
+import org.fenixedu.academic.domain.organizationalStructure.Unit;
+import org.fenixedu.academic.domain.organizationalStructure.UnitClassification;
+import org.fenixedu.bennu.core.annotation.DefaultJsonAdapter;
+import org.fenixedu.bennu.core.domain.Bennu;
+import org.fenixedu.bennu.core.json.JsonAdapter;
+import org.fenixedu.bennu.core.json.JsonBuilder;
+import org.fenixedu.commons.i18n.LocalizedString;
+import org.joda.time.YearMonthDay;
+
+import pt.ist.fenixframework.Atomic;
+import pt.ist.fenixframework.FenixFramework;
+import pt.utl.ist.fenix.tools.util.i18n.MultiLanguageString;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+@DefaultJsonAdapter(AdministrativeOffice.class)
+public class AdministrativeOfficeJsonAdapter implements JsonAdapter<AdministrativeOffice> {
+    @Override
+    public JsonElement view(AdministrativeOffice administrativeOffice, JsonBuilder ctx) {
+        JsonObject object = new JsonObject();
+        object.add("name", ctx.view(administrativeOffice.getName()));
+        object.addProperty("externalId", administrativeOffice.getExternalId());
+        object.add("space", ctx.view(administrativeOffice.getCampus()));
+        object.add("coordinator", ctx.view(administrativeOffice.getCoordinator()));
+        return object;
+    }
+
+    @Atomic
```

Comment:
The annotation is not needed here, as the JsonAdapter infrastructure will already provide the write transaction.